src/components/bucket/BucketEdit.js

```diff
@@ -7,17 +7,20 @@ import BucketTabs from "./BucketTabs";
 
 export default class BucketEdit extends Component {
   deleteBucket = (bid) => {
-    const {deleteBucket} = this.props;
+    const {deleteBucket, bucket} = this.props;
+    const {last_modified} = bucket;
     if (confirm("This will delete the bucket and all the collections and " +
                 "records it contains. Are you sure?")) {
-      deleteBucket(bid);
+      deleteBucket(bid, last_modified);
```

Comment:
I'm uncomfortable doing this in a view component; I suspect we could inspect the resource data state from the store in a saga to infer what should be sent to the server?